src/client/styles/index.scss

```diff
@@ -1,8 +1,22 @@
-h1 {
-  font-size: 96px;
-}
-
-body {
-  margin: 0 auto;
-  text-align: center;
-}
+/*
+ * Table of content
+ * 1. Global
+ * 2. Colors
+ * 3. Containers
+ * 4. Components
+ */
+
+// 1. Global
+@import './global/reset';
+
+// 2. Colors
+@import './variables/colors';
+
+// 3. Containers
+@import './containers/app';
+
+// 4. Components
+@import './components/Header';
```

Comment:
According to our experience, we have found that it is easier to maintain styles, when they are close to the components. So please consider moving component styles, closer to the components (e.g. in the same directory, as the component)

Reply:
We decided to leave it as it it right now.